src/connection.js

```diff
@@ -1126,6 +1133,18 @@ const ConfirmedTransactionMetaResult = struct.union([
   struct.pick({
     err: TransactionErrorResult,
     fee: 'number',
+    innerInstructions: struct.array([
+      struct({
+        index: 'number',
+        instructions: struct.array([
+          struct({
+            accounts: struct.array(['number']),
+            data: 'string',
+            programIdIndex: 'number',
+          }),
+        ])}
```

Comment:
We'll need the following struct for parsed inner instructions, what you have is fine for non-parsed

```rust
        instructions: struct.array([
          struct.union([
            struct({
              accounts: struct.array(['string']),
              data: 'string',
              programId: 'string',
            }),
            struct({
              parsed: 'any',
              program: 'string',
              programId: 'string',
            }),
          ]),
        ]),
```